File: codigos/1 - estrutura sequencial/1020 - Idade em Dias/Main.dart
import 'dart:io';

main(){
  int total = int.parse(stdin.readLineSync()!);
  
  int anos = total ~/ 365;
  int meses = (total - anos * 365) ~/ 30;
  int dias = total - anos * 365 - meses * 30;

  stdout.writeln("$anos ano(s)");
  stdout.writeln("$meses mes(es)");
  stdout.writeln("$dias dia(s)");
}

File: codigos/1 - estrutura sequencial/1020 - Idade em Dias/Main.cs
using System;
using static System.Console; 

// class URI {

    // static void Main(string[] args) { 

        int total = int.Parse(ReadLine());
        int anos = total / 365;
        int meses = (total - anos * 365) / 30;
        int dias = total - anos * 365 - meses * 30;

        WriteLine($"{anos} ano(s)");
        WriteLine($"{meses} mes(es)");
        WriteLine($"{dias} dia(s)");

    // }

// }